docs/components/status-notifications/status-counter.md

```diff
@@ -0,0 +1,31 @@
+
+# Status Counter
+
+The **status counter** component (formerly **icon status**) represents a customizable icon with a counter.
+
+## Code ---
+
+### Usage
+
+```ts
+import { SiStatusCounterComponent } from '@siemens/element-ng/status-counter';
+// SiIconStatusComponent is deprecated and kept as an alias for backward compatibility.
+@Component({
+  imports: [SiStatusCounterComponent, ...]
+})
+!!! warning "Deprecation Notice"
+  `SiIconStatusComponent` is deprecated and will be removed in a future release.  
+  Please use `SiStatusCounterComponent` instead.
+```
```

Comment:
@mistrykaran91 I don't think we need to mention any deprecations in the usage example here. We already have it in the code.